test-three-nathanpeter/menu/src/MenuBar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import {
+  MenuContainer,
+  DropBtn,
+  DropdownContent,
+  Dropdown,
+} from "./MenuBarStyles";
+
+import menu from "./data/menudata";
+
+const MenuBar = () => {
+  return (
+    <MenuContainer>
+      {menu.children.map((outerMap, i) => (
+        <Dropdown>
+          <DropBtn key={i}>
+            <h2>{outerMap.name}</h2>
+          </DropBtn>
+          <DropdownContent>
+            {outerMap.name === "Account" &&
```

Comment:
Can you explain why you're doing this instead of having something more generic?

Reply:
basically just because its a more modern way to conditional render inline using that && operator and I'm trying to make a pattern of using it more, just by embedding expressions within the jsx, everything on the left is just a check seeing if it's true and everything on the right of && will appear in the output